github/resource_github_membership_test.go

```diff
@@ -134,9 +157,21 @@ func testAccCheckGithubMembershipRoleState(n string, membership *github.Membersh
 	}
 }
 
-var testAccGithubMembershipConfig string = fmt.Sprintf(`
+func testAccGithubMembershipConfig(username string) string {
+	return fmt.Sprintf(`
   resource "github_membership" "test_org_membership" {
     username = "%s"
     role = "member"
   }
-`, testCollaborator)
+`, username)
+}
+
+func testAccGithubMembershipTheSame(orig, other *github.Membership) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if *orig.URL != *other.URL {
```

Comment:
should this also test to make sure at least one isn't null/empty? or should two empties be equal?

Reply:
I think, for the intent of the test, two empties would be equal.  I can't think of a time when this would be empty here, but if they were to both be empty, I think it means the case change didn't have an effect.